scripts/zones/Beadeaux_[S]/mobs/Da_Dha_Hundredmask.lua

```diff
@@ -0,0 +1,54 @@
+-----------------------------------
+-- Area: Beadeaux [S] (92)
+--  NM:  Da'Dha Hundredmask
+-- @spawnmob 17154195
+-- @pos -89.901 .225 -159.694 92
+-----------------------------------
+
+-- needs 2 hour perfect dodge
+
+-----------------------------------
+-- onMobSpawn
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:setMod(MOD_TRIPLE_ATTACK, 5);
+end;
+
+-----------------------------------
+-- onMobFight Action
+-----------------------------------
+
+function onMobFight( mob, target )
+    if (mob:getHPP() < math.random(5,65) and mob:getLocalVar("perfect_dodge") == 0) then
+        mob:useMobAbility(1013);
+		mob:setLocalVar("perfect_dodge", 1);
```

Comment:
Teo we used same scripting from Kirin, and with testing it worked fine, doesn't that random just make him use the ability at a random %?